repos/system_upgrade/common/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -217,7 +217,21 @@ def get_sysctls_status():
 
 def get_repositories_status():
     """ Get a basic information about YUM repositories installed in the system """
-    return RepositoriesFacts(repositories=repofileutils.get_parsed_repofiles())
+    try:
+        return RepositoriesFacts(repositories=repofileutils.get_parsed_repofiles())
+    except repofileutils.InvalidRepoDefinition as e:
+        raise StopActorExecutionError(
+            message='Invalid repository definition',
+            details={
+                'details': str(e),
+                'file': e.repofile,
+                'repoid': e.repoid,
+                'hint': 'To resolve this issue, please visit: {url}'
+                        .format(
+                            url='https://access.redhat.com/solutions/3185891'
+                        )
```

Comment:
The link in the hint seems to be incorrect (maybe a typo)?

Also the wording of the hint might be misleading for the users, visiting the link will to resolve the issue :), something like
`For more directions on how to resolve the issue, see <link>.` is more accurate.
Of course it depends on what is actually at the correct link.

Reply:
Changed the wording, and corrected the link.